src/main/resources/db/migration/V0_2__create_region_table.sql

```diff
@@ -0,0 +1,6 @@
+create extension if not exists "uuid-ossp";
```

Comment:
No need to repeat this here as it was already declared in the first migration script